src/XMLimport.h

```diff
@@ -50,7 +50,7 @@ class XMLimport : public QXmlStreamReader
 public:
     XMLimport(Host*);
 
-    bool importPackage(QIODevice* device, QString packageName = QString(), int moduleFlag = 0);
+    bool importPackage(QIODevice*, QString packageName = QString(), int moduleFlag = 0, QString* pVersionString = Q_NULLPTR);
```

Comment:
`Q_NULLPTR` resolves to `nullptr` as far I could see and when I started using it I wasn't coding exclusively for C++11 (e.g. recent [back-port to Qt4](https://github.com/SlySven/Mudlet/tree/release_30-qt4) - I was also using it as that seemed to be the _Qt way_.  What tools do not understand it BTW?